drizzlepac/haputils/align_utils.py

```diff
@@ -1318,3 +1318,113 @@ def register_photutils_function(name):
 
     func = getattr(background, name)  # raises AttributeError if not found
     return func
+
+# --------------------------------------------------------------------------------------------------------------
+########################################################################################
+# Author: Ujash Joshi, University of Toronto, 2017                                     #
+# Based on Octave implementation by: Benjamin Eltzner, 2014 <b.eltzner@gmx.de>         #
+# Octave/Matlab normxcorr2 implementation in python 3.5                                #
+# Details:                                                                             #
+# Normalized cross-correlation. Similiar results upto 3 significant digits.            #
+# https://github.com/Sabrewarrior/normxcorr2-python/master/norxcorr2.py                #
+# http://lordsabre.blogspot.ca/2017/09/matlab-normxcorr2-implemented-in-python.html    #
+########################################################################################
+
+import numpy as np
+from scipy.signal import fftconvolve
```

Comment:
Although it seems this code has been absorbed into the drizzlepac source from the quoted reference, I wonder if the imports should still all be at the top of the module to avoid duplication (import numpy as np) or "hidden" imports.